plugins/services/src/js/columns/ServicesTableStatusColumn.tsx

```diff
@@ -19,8 +19,28 @@ const StatusMapping: any = {
 export function statusRenderer(
   service: Service | Pod | ServiceTree
 ): React.ReactNode {
-  const serviceStatusText: string = service.getStatus();
-  const serviceStatusClassSet: string = StatusMapping[serviceStatusText] || "";
+  const serviceStatus = service.getStatus();
+  let serviceStatusClassSet: string = "";
+  let statusContent: JSX.Element | null = null;
+  if (serviceStatus.type === "servicetreestatussummary") {
```

Comment:
I was trying to log what's happening here and it looks like this condition is never met (even with a group in the services table). It always goes to the `else if` fallback, and as a result the entire object is set as the Trans id. This (somehow?) figures out how to be rendered correctly, even though `values` are nested inside of `id`. I'm not certain this would work in a prod build. I tried changing the condition to `if(typeof serviceStatus === "object")` and found that this condition allowed the id and values to be set as expected. Not sure if I'm explaining this right.
Inspecting the status with loop as is (id is an object):
<img width="355" alt="Screen Shot 2019-04-04 at 4 08 04 PM" src="https://user-images.githubusercontent.com/19582796/55594392-4a4bc780-56f4-11e9-8f61-22b6de11ae47.png">
Inspecting the status with typeof check:
<img width="346" alt="Screen Shot 2019-04-04 at 4 08 53 PM" src="https://user-images.githubusercontent.com/19582796/55594413-5768b680-56f4-11e9-97e2-9636c9bb1cd2.png">

Reply:
Weird I wonder why I didn’t see that. 

I’m fine with the object check here instead. I actually had that originally but then typescript complained about accessing `.id` etc. So I changed to this to resolve that. I wonder if it would be better to make ServiceTreeStatusSummary a class instead of an interface, then I can do an `instanceOf` check instead.

Reply:
Fixed, weird that it worked at all, but I guess `lingui/Trans` has guards for receiving an object with `id` & `values` as `id`